submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/web/model/SubmitWebRequest.kt

```diff
@@ -24,6 +25,7 @@ sealed class SubmitWebRequest(
     val format: SubFormat,
     val fileMode: FileMode,
     val files: List<File>,
+    val preferredSource: PreferredSource,
```

Comment:
same as before this class in my opinion are becoming like a map full of everything.